aws_google_auth/prepare.py

```diff
@@ -18,9 +18,10 @@ def default_if_none(value, default):
         return value if value is not None else default
 
     google_config.profile = default_if_none(profile, google_config.profile)
-
     _create_base_aws_cli_config_files_if_needed(google_config)
-    _load_google_config_from_stored_profile(google_config, google_config.profile)
+
+    if google_config.profile is not None:
```

Comment:
No I had it wrong, the `_load_google_config_from_stored_profile` is only needed when there is a profile to lookup.

For now this looks good.  In the future will look at some tests around this workflow to ensure it is solid.